packages/jellyfish-api-core/src/category/token.ts

```diff
@@ -98,6 +98,21 @@ export class Token {
   async mintTokens (amountToken: string, utxos: UTXO[] = []): Promise<string> {
     return await this.client.call('minttokens', [amountToken, utxos], 'number')
   }
+
+  /**
+   * Creates a transaction to burn tokens.
+   *
+   * @param {string} amounts Amount as json string, or array. Example: '[ \"amount@token\" ]'
+   * @param {string} from Address containing tokens to be burned
+   * @param {string} context Additional data necessary for specific burn type
+   * @param {UTXO[]} [utxos = []] A json array of json objects. Provide it if you want to spent specific UTXOs
+   * @param {string} [utxos.txid] The transaction id
+   * @param {number} [utxos.vout] The output number
+   * @return {Promise<string>} The hex-encoded hash of broadcasted transaction
+   */
+  async burnTokens (amounts: string, from: string, context?: string, utxos: UTXO[] = []): Promise<string> {
```

Comment:
Not tested?

Reply:
Seems like this was part of the cherry-pick I did on a10afe2657ee00ae0 from #1860. The tests are on [that PR](https://github.com/JellyfishSDK/jellyfish/pull/1860/files#diff-b78b933dcd14e178a555768bb95090dba23b139884641dc16ecb1c784e000fd6).